Edit features feature linked annotation (#377)
Esri/arcgis-maps-sdk-kotlin-samples: samples/edit-features-with-feature-linked-annotation/README.md

# Edit features with feature-linked annotation

> Edit feature attributes which are linked to annotations through an expression.

Reviewer: ```suggestion
Edit feature attributes which are linked to annotation through an expression.
```

## How to use the sample

Pan and zoom the map to see that the text on the map is annotation, not labels. Tap one of the address points to update the house number (`AD_ADDRESS`) and street name (`ST_STR_NAM`) and change its geometry. Tap one of the dashed parcel polylines and tap another location to change its geometry. **Note:** Selection is only enabled for points and straight (single segment) polylines.

The feature-linked annotation will update accordingly.

## How it works

1. Load the geodatabase. **Note:** Read/write geodatabases should normally come from a `GeodatabaseSyncTask`, but this has been omitted here.
2. Create `FeatureLayer`s from geodatabase feature tables found in the geodatabase.
3. Create `AnnotationLayer`s from geodatabase annotation tables found in the geodatabase.
4. Add the `FeatureLayer`s and `AnnotationLayer`s to the map's operational layers.
5. Use a map tap gesture to select address points or parcel polyline features. **Note:** Selection is only enabled for points and straight (single segment) polylines.

>    * For the address points, an alert is opened to allow editing of the address number (`AD_ADDRESS`) and street name (`ST_STR_NAM`) attributes.

Reviewer: ```suggestion
   * For the address points, a dialog is opened to allow editing of the address number (`AD_ADDRESS`) and street name (`ST_STR_NAM`) attributes.
```
